types/types.go

```diff
@@ -224,8 +224,8 @@ type Namespace struct {
 
 // DVOMetadata structure contains basic information about workload metadata
 type DVOMetadata struct {
-	Recommendations int         `json:"recommendations"`
-	Objects         int         `json:"objects"`
+	Recommendations uint        `json:"recommendations"`
+	Objects         uint        `json:"objects"`
```

Comment:
Just curious - might this change affect the data and they format received from kafka or the data stored/read from DB?

Reply:
Data moved through Kafka is not affected, as the JSON remains the same. This indicates at Go level that the integer is unsigned (so always positive). Same thing with the data stored in the DB, it is stored in the format specified in the database (INTEGER NOT NULL) as long as the data is a valid integer, which should not change.